client/landing/jetpack-cloud/components/activity-card/index.jsx

```diff
@@ -12,15 +14,36 @@ import { Card } from '@automattic/components';
 import ActivityActor from 'my-sites/activity/activity-log-item/activity-actor';
 import ActivityDescription from 'my-sites/activity/activity-log-item/activity-description';
 import { applySiteOffset } from 'lib/site/timezone';
+import PopoverMenu from 'components/popover/menu';
 
 /**
  * Style dependencies
  */
 import './style.scss';
+import downloadIcon from './download-icon.svg';
+import cloudIcon from './cloud-icon.svg';
 
 class ActivityCard extends Component {
+	constructor() {
+		super();
+		this.state = {
+			showPopoverMenu: false,
+		};
+	}
+
+	togglePopoverMenu = () => this.setState( { showPopoverMenu: ! this.state.showPopoverMenu } );
+	closePopoverMenu = () => this.setState( { showPopoverMenu: false } );
+	createRestoreUrl = restoreId => `/backups/${ this.props.siteSlug }/restore/${ restoreId }`;
+	createDownloadUrl = downloadId => `/backups/${ this.props.siteSlug }/download/${ downloadId }`;
+	createDetailUrl = backupId => `/backups/${ this.props.siteSlug }/detail/${ backupId }`;
+	triggerRestore = () => page.redirect( this.createRestoreUrl( this.props.activity.rewindId ) );
+	triggerDownload = () => page.redirect( this.createDownloadUrl( this.props.activity.rewindId ) );
+	triggerDetails = () => page.redirect( this.createDetailUrl( this.props.activity.rewindId ) );
```

Comment:
We could use the helper functions that Caleb implemented in `jetpack-cloud/sections/backups/paths.ts`

Reply:
- Paths fixed in https://github.com/Automattic/wp-calypso/pull/40867/commits/d1e1014faf87cd52c7a94ecb6c263c778737ccb6
- Styling fixed in https://github.com/Automattic/wp-calypso/pull/40867/commits/963c05e7d6e37618038ebaa2ef6bb18acf7c78a2

Thanks for the feedback :-)